staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -88,24 +89,24 @@ var (
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
 	// TODO(audit): figure out a general way to let the client choose their preferred version
-	groupVersions = []schema.GroupVersion{auditv1beta1.SchemeGroupVersion}
-	registry      = registered.NewOrDie("")
+	registry = registered.NewOrDie("")
 )
 
 func init() {
-	registry.RegisterVersions(groupVersions)
-	if err := registry.EnableVersions(groupVersions...); err != nil {
-		panic(fmt.Sprintf("failed to enable version %v", groupVersions))
+	allGVs := []schema.GroupVersion{auditv1alpha1.SchemeGroupVersion, auditv1beta1.SchemeGroupVersion}
```

Comment:
I'll repeat the question I had in the earlier PR. Is it possible to move this piece of code to the `NewBackend` method to activate only the version that was selected as preferred?